packages/management-console/webpack.common.js

```diff
@@ -11,7 +12,10 @@ module.exports = {
   plugins: [
     new HtmlWebpackPlugin({
       template: path.resolve(__dirname, 'src', 'index.html')
```

Comment:
@Sara4994 I found that all we need is to add the follwing:
```suggestion
      template: path.resolve(__dirname, 'src', 'index.html')
      favicon: 'src/favicon.ico'
    })
```

then rename the image to favicon.ico but keep it at the src level.
CopyWebpackPlugin will copy all images to image directory, which is not needed.

Reply:
@cristianonicolai Yes i tried this at first place, but still favicon wasnt showing up. So went with other aproach which worked fine.. anyway will try onemore time.

Reply:
cool, ok, I got it working fine with this approach so let me know if doesnt work for you.

Reply:
@cristianonicolai Modified.